buffer/buffer.mbt

```diff
@@ -13,8 +13,8 @@
 // limitations under the License.
 
 ///| Extensible buffer.
-#deprecated("Use type `T` instead")
-pub(all) typealias Buffer = T
+// #deprecated("Use type `T` instead")
+pub(all) typealias T as Buffer
 
```

Comment:
this has been deprecated long time ago, maybe we can just remove it

Reply:
the problem is, the `deprecated` attribute is not taking effect. So users have never received the deprecation warning